src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -208,18 +209,37 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
 
         function getNote(personId, date) {
             createNote("", personId, date);
+            $scope.warnMessage = "";
             tdprPersonService.getNote(personId, date).then(function(response) {
                 if(response.status === 200) {
                     $scope.noteContent = response.data;
                     $scope.temporaryContent = response.data.description;
                 }
             }, function(failure) {
-               Notification.warning({
-                    message: 'Something went wrong with getting your note.',
-                    delay: 2000});
+                Notification.warning({
+                   message: 'Something went wrong with getting your note.',
+                   delay: 2000});
                }
         )}
 
+       $scope.getNoteFromPreviousWeek = function() {
+            var previousDate = $filter('date')(getDayOfTheWeek(new Date(), $scope.relativeDayNumber - 7 ), "dd-MM-yyyy");
+            tdprPersonService.getNote(id, previousDate).then(function(response) {
+                 if(response.status === 200) {
+                    $scope.noteContent = response.data;
+                    $scope.temporaryContent = response.data.description;
+                    $scope.warnMessage = "Please remember to submit your note.";
+                    if(response.data.description === "") $scope.warnMessage = "There was no content last week.";
+                } else {
+                    $scope.warnMessage = "You didn't submit any notes last week.";
```

Comment:
The else branch will execute on any response code other than 200, including server errors, etc. It would be good to be more explicit here. For example -- display this message for a 404 code and a "Something went wrong"-ish message for other codes.